pkg/labels/labels_test.go

```diff
@@ -612,3 +612,7 @@ func TestBuilder_Labels(t *testing.T) {
 		}).Labels(),
 	)
 }
+
+func TestLabels_Bytes(t *testing.T) {
```

Comment:
This is over-testing, we don't actually care about the exact bytes this returns. PromQL tests already cover this.

Reply:
OK. I have removed this unit test.